Formula/freeswitch.rb

```diff
@@ -151,6 +151,10 @@ class Freeswitch < Formula
   end
 
   def install
+    on_macos do
```

Comment:
Could we have a comment here explaining what this is for?

Reply:
Would something like the headline message along with related upstream issue (https://github.com/signalwire/freeswitch/issues/1145) be sufficient?

---

I'm actually not sure of exact reason why this is required yet.

If I had to guess, I think it is due to `_XOPEN_SOURCE` being set by `freeswitch` in certain places, which changes value of `_POSIX_C_SOURCE`.

Perhaps due to order of `include`s, we hit the odd behavior where the same condition to set `NSIG` and access `NSIG` are not equal.

Reply:
Ah, macro voodoo. very nice. I guess at minimum a comment should give some indication of when a workaround can be removed, so a link to the upstream issue would help.

Reply:
Okay for the comment.

For actual fix, another alternative may be to set `_XOPEN_SOURCE`, which would do the opposite of `_DARWIN_C_SOURCE`, i.e. darwin macro would probably allow platform-specific behavior while open-source macro would use portable functions instead.